apps/machine/zynqmp_r5/platform_info.c

```diff
@@ -97,149 +97,9 @@ static struct remoteproc rproc_inst;
 extern int init_system(void);
 extern void cleanup_system(void);
 
-static int zynqmp_r5_a53_proc_irq_handler(int vect_id, void *data)
```

Comment:
IPI reg macros should also been moved to the remoteproc implementation file, struct remoteproc_priv() should definition should either move to remoteproc driver or define in the platform info header file

Reply:
done with rebased commits.